cmd/juju/storage/remove.go

```diff
@@ -83,6 +96,26 @@ func (c *removeStorageCommand) Init(args []string) error {
 
 // Run implements Command.Run.
 func (c *removeStorageCommand) Run(ctx *cmd.Context) error {
+	noWaitSet := false
+	forceSet := false
+	c.fs.Visit(func(flag *gnuflag.Flag) {
+		if flag.Name == "no-wait" {
+			noWaitSet = true
+		} else if flag.Name == "force" {
+			forceSet = true
+		}
+	})
+	if !forceSet && noWaitSet {
+		return errors.NotValidf("--no-wait without --force")
+	}
+	var maxWait *time.Duration
```

Comment:
Do we need this to be defaulted to 1min if no-wait wasn't set? I guess the thinking is that it's defaulted in the apiserver. Actually, that seems right to me. Just leaving this here since I've already written it. ;)